dashboards/nrm4sap/sap-dashboard.json

```diff
@@ -1356,8 +1356,10 @@
             },
             "nrqlQueries": [
               {
-                "accountId": 0,
-                "query": "FROM NR_SAP_JOB SELECT uniqueCount(JOB_NAME) as 'No. of Jobs' FACET SYS_ID, JobServerGroup, INSTANCE SINCE 1 day ago LIMIT MAX"
+                "accountIds": [
```

Comment:
Looks like we may just need to run the `sanitize-dashboards.ts` script that we have in our `utils/` folder.

If you wouldn't mind [running the script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) and pushing those change up, then we can approve and merge in this pr 👍🏻

Reply:
It's actually totally fine to have an array of account ids, as long as it is set to `[ 0 ]`!